app/views/posts/_form_template.html.erb

```diff
@@ -0,0 +1,23 @@
+<div>&nbsp</div>
```

Comment:
Remove this type of hacky stuff-- if you want whitespace, use padding and margin.